i18n/update_translations.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
Why are we using a python script for this?
Is it possible to use a javascript script here?